packages/api-explorer/src/Doc.jsx

```diff
@@ -43,7 +43,14 @@ class Doc extends React.Component {
     this.toggleAuth = this.toggleAuth.bind(this);
     this.hideResults = this.hideResults.bind(this);
     this.waypointEntered = this.waypointEntered.bind(this);
-    this.Params = createParams(this.oas);
+
+    const operation = this.getOperation()
+    const operationExtension =
+      operation[extensions.EXPLORER_ENABLED] !== undefined
+        ? operation
+        : this.oas
+
+    this.Params = createParams(operationExtension);
```

Comment:
Since we're supplying both `this.oas` and `this.getOperation()` into `this.Params`, this call to `createParams` can remain at `this.oas`.

And to help clean up all the different parameter widgets, what you can probably do instead is create a new prop on the `Params` component called `explorerEnabled`. Default it to `true`, and then here in `Doc` when we do `<this.Params ... />`, supply that prop with your logic here. Then in the different widget components, you can clean it up to just look at `explorerEnabled`.

Doing this should allow you to clean up a lot all of the repeated `const explorerEnabled = oas[extensions.EXPLORER_ENABLED];` code that we've got there.